docs/differences/numerics.rst

```diff
@@ -0,0 +1,49 @@
+.. _numerics:
+
+In Python 3.x, floating point numbers are implemented using double 
+precision. This provides roughly 53 significant bits (17 digits or so)
+which means that many programmers can often (but not always) simply
+ignore the issue of limited floating-point precision without much
+risk of being caught out. The cost of putting floating point numbers
+into the heap is considered to be high enough that the slightly 
+higher performance that single precision numbers would provide is
+reckoned to be immaterial.
+
+MicroPython, on the other hand, typically runs on much lower performance
+hardware that typically lacks double precision floating point hardware
+and is commonly memory-constrained. There is also the perception that
+the microcontrollers that are typically used with MicroPython typically
```

Comment:
If the microcontroller includes 32-bit floating point hardware, MicroPython uses this by default on grounds of speed. It is usually possible to specify 64-bit software FP as a build option to prioritise precision over performance.